app/styles/app/layouts/log.scss

```diff
@@ -246,13 +246,13 @@
       position: absolute;
       padding-right: 1em;
     }
```

Comment:
```suggestion
      -webkit-user-select: none;
      -moz-user-select: none;
      -ms-user-select: none;
      user-select: none;
    }
```

Reply:
@piotr-travisci this is a great suggestion! Though, prefixes are added automatically by [autoprefixer](https://github.com/travis-ci/travis-web/blob/master/package.json#L34) and thus not necessary.

Reply:
👍 @artursmirnov.
Suggestion refactored.

Reply:
Thanks!